source/educators/navigation/content_creation_management.rst

```diff
@@ -211,7 +211,7 @@ initiative.
    :glob:
    
    ../concepts/instructional_design/libraries.rst
-   ../how-tos/course_development/create_library.rst
+   ../how-tos/course_development/create_legacy_library.rst
```

Comment:
I think this change can be reverted, since you didn't rename the page. But you will need to put the new page somewhere in the nav tree.